usage/rest-client/src/main/java/brooklyn/util/http/BuiltResponsePreservingError.java

```diff
@@ -62,7 +64,8 @@ public BuiltResponsePreservingError(int status, Headers<Object> headers, Object
             Exceptions.propagateIfFatal(e);
             return new BuiltResponsePreservingError(status, headers, entity, new Annotation[0], e);
         } finally {
-            source.close();
+            if (source instanceof BaseClientResponse)
```

Comment:
Why only call `close()` if it's an instance of `BaseClientResponse`? The close method is on `javax.ws.rs.core.Response`, so should we always call close?